src/common/components/contactInfo/contactInfo.component.js

```diff
@@ -117,7 +117,7 @@ class Step1Controller {
   }
 
   loadRadioStations () {
-    const postalCode = this.donorDetails.mailingAddress?.postalCode;
+    const postalCode = this.donorDetails.mailingAddress.postalCode
```

Comment:
This commit (9fc87f7685e69f8296553ee954c2e4aa46af79c7) should be broken up and squashed with the commits that introduced the lint errors.